services/apollo/Dockerfile

```diff
@@ -1,15 +1,30 @@
-FROM node:10.15.0
-ARG VERSION
-ENV PREFECT_SERVER_VERSION=${VERSION}
+ARG NODE_VERSION=${NODE_VERSION:10.15.0}
+FROM node:${NODE_VERSION}-slim
 
-COPY . /apollo
+# Prefect Version, default to MASTER
+ARG PREFECT_VERSION
+ENV PREFECT_VERSION=${PREFECT_VERSION:-master}
 
-RUN \
-  cd /apollo \
-  && npm ci && npm run build
+# Prefect Server Version, default to MASTER
+ARG PREFECT_SERVER_VERSION
+ENV PREFECT_SERVER_VERSION=${PREFECT_SERVER_VERSION:-master}
+
+ARG RELEASE_TIMESTAMP
+ENV RELEASE_TIMESTAMP=$RELEASE_TIMESTAMP
+
+# Image Labels
+LABEL maintainer="help@prefect.io"
+LABEL org.label-schema.schema-version = "1.0"
+LABEL org.label-schema.name="apollo"
+LABEL org.label-schema.url="https://www.prefect.io/"
+LABEL org.label-schema.version=${PREFECT_VERSION}
+LABEL org.label-schema.build-date=${RELEASE_TIMESTAMP}
 
 WORKDIR /apollo
-COPY post-start.sh .
-RUN chmod +x post-start.sh
+COPY . .
+
+RUN npm install && \
```

Comment:
Is there a reason you changed from `npm ci` to `npm install`?  I think we should stick with `npm ci` to ensure no surprising changes make their way into the image

Reply:
Agree, it should be ```npm ci```